.github/workflows/lint_pr_title.yml

```diff
@@ -0,0 +1,22 @@
+# DESCRIPTION:
+#  A workflow used to verify if PR titles matches conventional commits strategy.
+# END
+
+name: Lint PR Title
+run-name: "Lint PR - (#${{ github.event.number }}) ${{ github.event.pull_request.title }}"
+
+permissions:
+  pull-requests: read
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - ready_for_review
+
+jobs:
+  lint_pr_title:
+    name: Lint PR
+    uses: PaloAltoNetworks/terraform-modules-vmseries-ci-workflows/.github/workflows/lint_pr_title.yml@plan-apply
```

Comment:
I think it would be clear/better to include the step directly here instead of referencing ?